tests/rest-api.test.js

```diff
@@ -0,0 +1,246 @@
+const chai = require('chai');
+const expect = chai.expect;
+const should = chai.should();
+const nock = require('nock');
+
+const response = require('./rest-response');
+const restAPI = require('../8dev_restapi.js');
+
+describe('Rest API interface', () => {
+  
+  const url = 'http://localhost:8888';
+  const device_name = 'threeSeven';
+  const path = '/3312/0/5850';
+  const service = new restAPI.Service({ host: url});
+  const device = new restAPI.Device(service, device_name);
+  const tlvBuffer = new Buffer([0xe4, 0x16, 0x44, 0x00, 0x00, 0x00, 0x01]);
+
+  describe('Endpoint interface', () => {
+    
+    it('getObjects function should return an array of all endpont\'s resource paths', () => {
```

Comment:
Nerašyta praktika yra kad testo aprašymas prasidėtų `it('should ...`, o visi testai grupuojami po `describe`. Čia irgi būtų neblogai sustackinti `describe` viena po kitu:

```javascript
describe('Endpoint interface' ....
  describe('getObjects function', ...
    it('should return ....', ...

  describe('read() function', ...
    it('should do xxx', ...
`